doc/ref_model/chapters/chapter06.md

```diff
@@ -3,49 +3,49 @@
+In addition to the raw or base-level Cloud Infrastructure functionality to API and object mapping, it is further the intent to specify an explicit, normalized set of APIs and mappings to control the logical interconnections and relationships between these objects, notably, but not limited to, support of SFC (Service Function Chaining) and other networking and network management functionality.
 
-Chapter 3 introduced a model of the Network Function Virtualisation Infrastructure (NFVI). Figure 3-1 shows an overview of the NFVI model including the external application programming interface (API)/ user interface (UI) for providing access to the NFVI management functions. Section 3.3 lists the actions supported by the NFVI Management Software.  This chapter specifies the abstract interfaces (API, CLI, etc.) supported by the NFVI Reference Model. The purpose of this chapter is to define and catalogue a common set of open (not proprietary) APIs, of the following types:
+Chapter 3 introduced a model of the Cloud Infrastructure. Figure 3-1 shows an overview of the Cloud Infrastructure model including the external application programming interface (API)/ user interface (UI) for providing access to the Cloud Infrastructure management functions. Section 3.3 lists the actions supported by the Cloud Infrastructure Management Software.  This chapter specifies the abstract interfaces (API, CLI, etc.) supported by the Cloud Infrastructure Reference Model. The purpose of this chapter is to define and catalogue a common set of open (not proprietary) APIs, of the following types:
 
-- NFVI APIs: These APIs are provided to the VNF workloads (i.e. exposed), by the infrastructure in order for VNF workloads to access (i.e. request, consume, control, etc.) NFVI resources.
-- Intra-NFVI APIs: These APIs are provided and consumed directly by the infrastructure. These APIs are purely internal to the NFVI and are not exposed to the workloads.
-- Enabler Services APIs: These APIs are provided by non-NFVI services and provide capabilities that are required for a majority of VNF workloads, e.g. DHCP, DNS, NTP, DBaaS, etc.
+- Cloud Infrastructure APIs: These APIs are provided to the workloads (i.e. exposed), by the infrastructure in order for workloads to access (i.e. request, consume, control, etc.) Cloud Infrastructure resources.
+- Intra-Cloud Infrastructure APIs: These APIs are provided and consumed directly by the infrastructure. These APIs are purely internal to the Cloud Infrastructure and are not exposed to the workloads.
+- Enabler Services APIs: These APIs are provided by non-Cloud Infrastructure services and provide capabilities that are required for a majority of VNF workloads, e.g. DHCP, DNS, NTP, DBaaS, etc.
```

Comment:
Was the inclusion of VNF deliberate here?  

```suggestion
- Enabler Services APIs: These APIs are provided by non-Cloud Infrastructure services and provide capabilities that are required for a majority of workloads, e.g. DHCP, DNS, NTP, DBaaS, etc.
```

Reply:
Fine with proposed change. We don't need VNF here.

Reply:
Corrected